user_templates/mcu_defaults/main/overrides/BTT_SKR_Pico_v1.0_overrides.cfg

```diff
@@ -0,0 +1,27 @@
+
+#-------------------------#
+#   MCU board overrides   #
+#-------------------------#
+
+# Those overrides are automatically added when you select a mcu board with specific configuration options like shared pins.
```

Comment:
As discussed in the other PR. It's a good idea to add this directly to the MCU file. This way there is no need to handle the specific boards overrides and it will be much simpler for me in the install script (i'm going to make it evolve soon as discussed).

Reply:
this would mean that extruder would also get in mcu.cfg which is basically often not used in certain cases. so i am not really happy about ripping the "special" config apart in multiple files, instead of everything special in overrides.cfg

Reply:
I would say extruder no, but TMC definition yes it could be moved here.

In fact what I want to move to mcu.cfg is all the definition of "pins" that need to be mapped not to the main MCU but to the toolhead mcu. Current and things like that can stay in overrides.cfg